clang/lib/StaticAnalyzer/Checkers/PointerArithChecker.cpp

```diff
@@ -74,6 +74,26 @@ class PointerArithChecker
 
 REGISTER_MAP_WITH_PROGRAMSTATE(RegionState, const MemRegion *, AllocKind)
 
+namespace {
```

Comment:
We follow the LLVM style guide, which promotes the keyword static. Anon namespaces should be used for class declarations though, but not bundling multiple decls into it usually.